src/app/components/tabs/index.html

```diff
@@ -25,6 +25,11 @@
       The text of the button that selects the tab.
     </sky-demo-page-property>
     <sky-demo-page-property
+        propertyName="tabHeaderCount"
+    >
+      (Optional.) Displays a counter alongside the tab header.
```

Comment:
The optional part of this is a flag they have on their sky-demo-page-property components. Look at the alert's index.html for an example.